CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## 1.3.0
```

Comment:
Version must be the same as specified in legacy visual.

Reply:
@ignatvilesov why?

Reply:
Legacy version is `1.3.4`. Change log shows `1.3.0`. That's a bit confused.

The version must be the same as a legacy or greater.